test/manual-scenarios/swap/smart transactions.csv

```diff
@@ -0,0 +1,17 @@
+Step,Test steps,Preconditions,Test data,Expected result,Notes
```

Comment:
I'm thinking about the file naming :thinking: So here we have:

- Swap with Smart Transaction  -> here we call this case `smart transactions`
- Swap without Smart Transaction -> here we call this case `swap eth`

Maybe something like this could be more explanatory?
- `swap with smart transaction`
- `swap without smart transaction`

or
- `swap`
- `smart swap`

saying this because Smart Transactions also exist outside Swaps, (Send Eth + Smart Transaction)
what do you think?

Reply:
Ah yes very good point. Smart swap makes sense when you put it like this (this is how it was called previously) but smart swap being taken out of swap settings and changed to smart transactions, it made more sense to name it that way too to avoid confusion. But I think the first option makes more sense and is clearer.